src/main/scala/com/amazon/deequ/analyzers/QuantileNonSample.scala

```diff
@@ -0,0 +1,307 @@
+package com.amazon.deequ.analyzers
+
+import scala.collection.immutable.ListMap
+import scala.collection.mutable.ArrayBuffer
+import scala.reflect.ClassTag
+import scala.util.control.Breaks._
+
+
+class QuantileNonSample[T](val sketchSize: Int,
+                           val shrinkingFactor: Double = 0.64)
+                          (implicit ordering: Ordering[T],
+                           ct: ClassTag[T]) extends Serializable{
+
+  private var curNumOfCompactors = 0
+  // number of items in compactors
```

Comment:
Please use /**/ and have comments start with capital letters

Reply:
Done.